Azure/sonic-buildimage
# [device] Upgrade as7726-32x complying the BCM SAI 3.5.3.1

Signed-off-by: derek_sun derek_sun@edge-core.com

- What I did
Modify SDK configuration and add LED control binary files.

- How I did it
Change SDK configuration file, config file path and add LED control binary files.

- How to verify it
1)Jenkins r138, 2019-11-26 image to verify/test/confirm with modify files.
2)rebuilding SONiC.

- Description for the changelog
Change configuration file, config file path and LED control binary files.

- A picture of a cute animal (not mandatory but encouraged)
